TACOS-persistence/src/main/java/cz/fi/muni/TACOS/persistence/dao/AbstractDao.java

```diff
@@ -23,6 +23,7 @@ public void create(T entity) {
             throw new IllegalArgumentException("Parameter entity cannot be null!");
         }
         em.persist(entity);
+        em.flush();
```

Comment:
I think we should not use the flush just because the tests need it. I would suggest remove it everywhere.